experiments/yarn/src/main/resources/reference.conf

```diff
@@ -4,7 +4,7 @@ gearpump {
   ###########################
   yarn {
     client {
-      package-path = "/user/gearpump/gearpump.zip"
+      package-path = "/usr/lib/gearpump/gearpump-${version}.zip"
```

Comment:
/usr/lib/gearpump/gearpump-${version}.zip is wrong

It will try to resolve this file in HDFS.